apps/examples/tls_client/tls_client_main.c

```diff
@@ -1339,6 +1355,42 @@ int tls_client_cb(void *args)
 	 */
 	retry_left = opt.max_resend;
 send_request:
+		if (opt.aging > 0) {
+		mbedtls_printf("   start the aging test (%d)B sendbuf size(%d)B\n", opt.aging, sizeof(buf));
+		int send_left = opt.aging;
+
+		// send buf size
+		unsigned int aging_size = htonl(opt.aging);
+		ret = mbedtls_ssl_write(&ssl, (void *)&aging_size, sizeof(unsigned int));
+		if (ret <= 0 && ret != MBEDTLS_ERR_SSL_WANT_READ &&
+			ret != MBEDTLS_ERR_SSL_WANT_WRITE) {
+			mbedtls_printf(" failed to send transfer size to the tls server(%x)\n", -ret);
+			goto exit;
+		}
+		
+		while (send_left > 0) {
+		    int datasize = send_left < MBEDTLS_SSL_MAX_CONTENT_LEN? send_left : MBEDTLS_SSL_MAX_CONTENT_LEN;
+			for(written = 0, frags = 0; written < datasize; written += ret, frags++ )
```

Comment:
1. need space after for
2. remove space before closing braket
3. put opening bracket at end of line
```
for (xxxx) {
```